web/fns.php

```diff
@@ -59,37 +59,57 @@ function make_request($url, $token, $params) {
     return $data;
 }
 
-function alloy_process_page($token, $design, $bbox, $page, $join_attributes=null, $join_function=null) {
+function alloy_process_page($token, $design, $bbox, $page) {
+    $k = get('url', 'https://[a-z.]+');
+    $cfg = ALLOY_LAYER_CONFIG[$k][$design];
```

Comment:
Crashes on L105 if there's no custom config for this layer, this fixes:

```suggestion
    $cfg = ALLOY_LAYER_CONFIG[$k][$design] ?? [];
```